unzip_content.py

```diff
@@ -287,11 +297,19 @@ def get_icon_path(linkJSON):
                     url = link_data['linkUrl']
                     desktop_objects[id].append_localized_url(lang, url)
 
-    apps_path = os.path.join(UNZIP_DIR, 'apps', 'content.json')
+    apps_path = os.path.join(CONTENT_DIR, 'apps', 'content.json')
     apps_file = open(apps_path)
     apps_json = json.load(apps_file)
     apps_file.close()
     for app_data in apps_json:
+        # For the desktop files, we want to prefix and custom Endless
+        # categories with "X-Endless-"
+        categories = app_data['category'].split(';')
+        for i, category in enumerate(categories):
+            if category in X_ENDLESS_CATEGORIES:
+                categories[i] = 'X-Endless-' + category
```

Comment:
This is necessary, because the appstream XML does not use the "X-Endless-" prefix.  And, while I could in theory specify as "X-Endless-Family" all over the place and then strip out the "X-Endless-" prefix when generating the data for the appstream XML, it would be ugly (and possibly error-prone) to have to keep that prefix everywhere.  At the moment, the only category that gets the "X-Endless-" prefix in the desktop files is "Family".  Other categories are left untouched.